.github/workflows/ci-build.yml

```diff
@@ -164,8 +168,21 @@ jobs:
         driver: docker-container
         use: true
     - name: Build
+      id: bake
       uses: docker/bake-action@v4.0.0
       with:
         files: docker-bake.hcl
         workdir: .
-        targets: interop_binaries
+        targets: interop_binaries_small
+        load: true
+        set: |
+          janus_interop_client_small.tags=janus_interop_client
+          janus_interop_aggregator_small.tags=janus_interop_aggregator
+          janus_interop_collector_small.tags=janus_interop_collector
+    - name: Save tarball
```

Comment:
I think I'm confused -- AFAICT the steps are:
1) Load existing interop-binary-images from artifact store (if available)
2) Build & run tests (but don't build containers)
3) Build & store containers

This runs container tests against whatever was saved last, which might be from an arbitrary PR (which may or may not be really functional). I think we should test against containers built against the current PR, which I think would let us not save the images at all.

Is the intent to test against a known-working container image? If so, we should use a tagged & published container image, IMO.